code/game/objects/items/explosives/grenades/marines.dm

```diff
@@ -362,3 +363,9 @@ proc/flame_radius(radius = 1, turf/T, burn_intensity = 25, burn_duration = 25, b
 		if(launched && CHECK_BITFIELD(resistance_flags, ON_FIRE))
 			var/armor_block = L.run_armor_check(target_zone, "fire")
 			L.apply_damage(rand(throwforce*0.75,throwforce*1.25), BURN, target_zone, armor_block) //Do more damage if launched from a proper launcher and active
+
+	// Flares instantly burn out nodes when thrown at them.
+	var/obj/effect/alien/weeds/node/N = locate() in loc
+	if(N && istype(N))
+		N.fire_act()
+		turn_off()
```

Comment:
Will it consume no fuel?

Reply:
`turn_off()` sets fuel and active to `0` / `FALSE`.

Reply:
So it will immediately snuff out the flare. Alright.